src/claudesavvy/web/templates/pages/harness.html

```
@@ -0,0 +1,295 @@
+                            </div>
+                            <span class="text-sm font-semibold text-gray-800">{{ category }}</span>
+                            <span class="ml-auto text-xs font-mono text-gray-400">{{ items | length }}</span>
+                        </div>
+                        <ul class="divide-y divide-gray-50">
+                            {% for item in items[:5] %}
+                            <li class="px-4 py-2.5 flex items-start gap-2">
+                                <span class="w-1.5 h-1.5 rounded-full mt-1.5 shrink-0" style="background: {{ color }};"></span>
+                                <div class="min-w-0">
+                                    <div class="text-xs font-semibold text-gray-800 truncate">{{ item.name }}</div>
+                                    {% if item.description %}
+                                    <div class="text-xs text-gray-400 truncate">{{ item.description[:80] }}{% if item.description|length > 80 %}…{% endif %}</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)